libphobos/libdruntime/core/sys/posix/stdio.d

```diff
@@ -279,7 +279,7 @@ unittest
 { /* Note: open_memstream is only useful for writing */
     import core.stdc.string : memcmp;
     char* ptr = null;
-    char[] testdata = ['h', 'e', 'l', 'l', 'o', 0];
+    char[6] testdata = ['h', 'e', 'l', 'l', 'o', 0];
```

Comment:
lib64:

| Type          | Bytes         |
| ------------- |:-------------:|
| phobos old    | 58822074      |
| phobos new    | 52352534      |
| druntime old  | 6592452       |
| druntime new  | 7387742       |

lib32:

| Type          | Bytes         |
| ------------- |:-------------:|
| phobos old    | 35083870      |
| phobos new    | 31351722      |
| druntime old  | 3818140       |
| druntime new  | 4369586       |


Hello World, unstripped:

| Type          | Bytes         |
| ------------- |:-------------:|
| old    | 7234800      |
| new    | 7233184      |

Hello World, stripped:

| Type          | Bytes         |
| ------------- |:-------------:|
| old    | 1140608      |
| new    | 1138992      |

The druntime library is a slightly bigger, as expected. The new libphobos doesn't embed libdruntime, so it's of course smaller.

The Hello World executable is even a little bit smaller with the new Makefiles. Druntime even compiles all sources (even sys/windows on linux machines!) so I think our current approach should be fine.